app/src/main/java/com/lightricks/feedexercise/database/FeedDatabase.kt

```diff
@@ -1,5 +1,21 @@
 package com.lightricks.feedexercise.database
 
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+
 /**
  * todo: add the abstract class that extents RoomDatabase here
  */
+
+@Database(entities = arrayOf(FeedItemEntity::class), version = 1)
+abstract class AppDatabase : RoomDatabase() {
+    abstract fun feedItemDao(): FeedItemDao
+
```

Comment:
Let's make it a Singleton.

First, declare the single instance:
`private lateinit var INSTANCE: AppDatabase`

Then, make the accessor method:
```
fun getDatabase(context: Context): FeedDatabase {
    synchronized(FeedDatabase::class.java) {
        if (!::INSTANCE.isInitialized) {
            // todo: create the instance here
        }
    }

    return INSTANCE
}
```